resources/src/js/router/views/signup/index.vue

```diff
@@ -6,68 +6,163 @@
+              accept="image/*"
+              @change="onFilePicked"
+            >
+            <v-btn color="primary" @click="submit">Submit</v-btn>
+          </v-flex>
+        </v-layout>
+      </v-form>
     </v-container>
   </v-content>
 </template>
 
 <script>
-import { required } from 'vuelidate/lib/validators';
+import { validationMixin } from "vuelidate";
+import {
+  alpha,
+  alphaNum,
+  email,
+  minLength,
+  numeric,
+  required,
+  sameAs
+} from "vuelidate/lib/validators";
+import axios from "axios";
+import { resolve } from "q";
+
 export default {
-  data: () => ({
-    organization: null,
-    firstName: null,
-    lastname: null,
-    email: null,
-    password: null,
-    validatepassword: null,
-    address1: null,
-    address2: null,
-    city: null,
-    country: null,
-    zip: null
-  }),
+  mixins: [validationMixin],
   validations: {
-      form: {
-          organization: { required },
-          firstName: { required },
-          lastName: { required },
-          email: { required },
-          password: { required },
-          validatePassword: { required },
-          address1: { required },
-          address2: { required },
-          city: { required },
-          country: { required },
-          zip: { required }
+    form: {
+      organization_name: { alphaNum, required },
+      first_name: { alpha, required },
+      last_name: { alpha, required },
+      email: { email, required },
+      password: { minLength: minLength(6), required },
+      password_confirmation: { sameAsPassword: sameAs("password"), required },
+      address_number: { numeric, required },
+      street: { alpha, required },
+      city: { alpha, required },
+      country: { alpha, required },
+      zip_code: { numeric, required },
+      state: { alpha, required }
+    }
+  },
+  data() {
+    return {
+      showPassword: false,
+      organization_name: null,
+      first_name: null,
+      last_name: null,
+      email: null,
+      password: null,
+      password_confirmation: null,
+      address_number: null,
+      street: null,
+      city: null,
+      country: null,
+      zip_code: null,
+      state: null,
+      imageName: null,
+      imageUrl: null,
+      imageFile: null
+    };
+  },
+  computed: {},
+  methods: {
+    pickFile() {
+      this.$refs.image.click();
+    },
+    onFilePicked(e) {
+      const files = e.target.files;
+      if (files[0] !== undefined) {
+        this.imageName = files[0].name;
+        if (this.imageName.lastIndexOf(".") <= 0) {
+          return;
+        }
+        const fr = new FileReader();
+        fr.readAsDataURL(files[0]);
+        fr.addEventListener("load", () => {
+          this.imageUrl = fr.result;
+          this.imageFile = files[0];
+        });
+      } else {
+        this.imageName = "";
+        this.imageFile = "";
+        this.imageUrl = "";
       }
+    },
+    submit() {
+      axios
+        .post("/api/register_admin", {
```

Comment:
So if you move all your data properties into a form object, you can just pass the form object in, like 
`axios.post('/api/register_admin', this.form);`